awx/conf/settings.py

```diff
@@ -171,7 +171,6 @@ def __init__(self, cache, registry, encrypter=None, decrypter=None):
     def get(self, key, **kwargs):
         value = self.cache.get(key, **kwargs)
         value = self._handle_encryption(self.decrypter, key, value)
-        logger.debug('cache get(%r, %r) -> %r', key, empty, filter_sensitive(self.registry, key, value))
```

Comment:
This ends up getting called a *lot*, and _itself_ does *more* `settings.LOG_...` lookups.